docs/source/CVAOpt_Algorithm/Parameters_and_Hyperparameters.md

```diff
@@ -42,3 +42,50 @@ The number of parents in a iteration
 
 The number of offsprings created in a iteration
 
+## Example
```

Comment:
this is exactly the notebook code from cvaopt, right?
Could we maybe instead of having this code in here just reference the notebook? So we only have to adapt one place? Or was this a specific requirement from @K1mpossible ?